xbmc/cores/VideoPlayer/MediaPipelineWebOS.cpp

```diff
@@ -520,12 +530,48 @@ bool CMediaPipelineWebOS::Load(CDVDStreamInfo videoHint, CDVDStreamInfo audioHin
   std::scoped_lock audioLock(m_audioCriticalSection);
 
   CVariant p;
-  CVariant payloadArgs;
 
   if (videoHint.cryptoSession || audioHint.cryptoSession)
   {
-    CLog::LogF(LOGERROR, "CryptoSessions unsupported");
-    return false;
+    CryptoSessionSystem keySystem = CRYPTO_SESSION_SYSTEM_NONE;
+
+    const int svpVersion = m_webOSVersion < 6 ? SVP_VERSION_40 : SVP_VERSION_30;
```

Comment:
Sorry, I think I made a mistake here when making the suggestion. This should be reversed and I think webOS 5 can use version 40 too.